OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -0,0 +1,308 @@
+
+```
+
+*Reuse the WebCenter Portal database schema*
+
+Update the following parameter inside `webcenter.env.list` :
+
+```
+DB_DROP_AND_CREATE=false
+
+```
+
+*Create container pointing to existing domain*
+*Run the command to create Container in given Sequence (run as non root user)*
+```
+# create Admin Container
+$ docker run -i -t --name WCPAdminContainer --network=WCPortalNET -p 7001:7001 -v /scratch/wcpdocker/volumes/wcpportal:/u01/oracle/user_projects --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/configureOrStartAdminServer.sh; /bin/bash"
+ 
+# create Portal Container
+$ docker run -i -t --name WCPortalContainer --network=WCPortalNET -p 8888:8888 -v /scratch/wcpdocker/volumes/wcpportal:/u01/oracle/user_projects --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterPortal.sh; /bin/bash"
+
+# create Elasticsearch Container
+$ docker run -i -t --name ESContainer --network=WCPortalNET -p 9200:9200 --volumes-from WCPortalContainer -v /scratch/wcpdocker/volumes/es:/u01/esHome/esNode/data --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/configureOrStartElasticsearch.sh; /bin/bash"
+
+```
+ 
+
+
+##### 2 How do I create containers with new domain?
+In case of any error or want a new fresh instance then you need to follow given instruction  in sequence:
+
+*Delete existing container (WCPAdminContainer, WCPortalContainer and ESContainer)*
+
+
+*Delete data from shared data volume*
+
+```
+# below command should be run as root user
+$ rm -rf  $ES_DATA_VOLUME/*
+$ rm -rf $DATA_VOLUME/*
+
+```
+
+*Drop and recreate the WebCenter Portal database schema*
+ 
+```
+$ docker container stop ESContainer
+$ docker container stop WCPortalContainer
+$ docker container stop WCPAdminContainer
+
+$ docker container rm ESContainer
+$ docker container rm WCPortalContainer
+$ docker container rm WCPAdminContainer
+
+
+```
+
+Update the following parameter inside `webcenter.env.list` :
+
+```
+DB_DROP_AND_CREATE=true
+
+```
+
+*Create containers*
+
+*Run the command to create Container in given Sequence (run as non root user)*
+
+```
+# create Admin Container
+$ docker run -i -t --name WCPAdminContainer --network=WCPortalNET -p 7001:7001 -v /scratch/wcpdocker/volumes/wcpportal:/u01/oracle/user_projects --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/configureOrStartAdminServer.sh; /bin/bash"
+ 
+# create Portal Container
+$ docker run -i -t --name WCPortalContainer --network=WCPortalNET -p 8888:8888 -v /scratch/wcpdocker/volumes/wcpportal:/u01/oracle/user_projects --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterPortal.sh; /bin/bash"
```

Comment:
Remove `/bin/bash` at least, but everything after the image name should really be removed.